python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -226,6 +228,14 @@ def resolve_assetMaterializations(
             )
         ]
 
+    # TODO: Prob want a more efficient way of resolving this
+    def resolve_configField(self, _graphene_info) -> Optional[GrapheneConfigTypeField]:
+        op = self.resolve_op(_graphene_info)
```

Comment:
I think we should try to pull the functionality of `resolve_op` into a helper function and then call the helper function to get back the op definition. In general, field resolvers of the same level should be independent of each other (so should not call one another).

Reply:
Good call, I've now done this.